tsl/src/continuous_aggs/options.c

```diff
@@ -284,3 +286,26 @@ continuous_agg_update_options(ContinuousAgg *agg, WithClauseResult *with_clause_
 		elog(ERROR, "cannot alter finalized option for continuous aggregates");
 	}
 }
+
+void
+warn_if_hierarchical_realtime_cagg(int32 parent_mat_hypertable_id, bool materialized_only)
+{
+	if (materialized_only)
+		return;
+
+	ContinuousAgg *parent_cagg;
+	int32 mat_hypertable_id = parent_mat_hypertable_id;
```

Comment:
Nit. 

Just for curiosity, do we need an extra variable here? Can't we live just with `parent_mat_hypertable_id` and set it bellow with the `parent_cagg->data.parent_mat_hypertable_id` value???

Reply:
I've added it for clarity; so it will not use a parameter as a variable;
it has the benefit that the original values can be used later in the function without the need to read it thru to check that is it being changed or not - in the scope of this function this may not seem that beneficial.

the compiler throws away the extra variable - this is purely just syntactic sugar

I've removed it